apps/meteor/client/views/marketplace/AppDetailsPage/tabs/AppDetails/AppDetails.tsx

```diff
@@ -9,6 +10,18 @@ import ScreenshotCarouselAnchor from '../../../components/ScreenshotCarouselAnch
 import type { AppInfo } from '../../../definitions/AppInfo';
 import AppDetailsAPIs from './AppDetailsAPIs';
 
+const normalizeUrl = (url: string): string => {
```

Comment:
What if `url` is not a valid linkable text?

Reply:
return a empty string...

Reply:
These lines contradict it: https://github.com/RocketChat/Rocket.Chat/pull/30012/files#diff-46795d9738beab48b68302a1312498cf6e19a77d2efd1a6981f5155e5e632716R20-R26